kolena/_utils/krequests.py

```diff
@@ -65,6 +65,7 @@ def _with_default_kwargs(**kwargs: Any) -> Dict[str, Any]:
         "Authorization": f"Bearer {client_state.jwt_token}",
         "X-Request-ID": uuid.uuid4().hex,
         "User-Agent": user_agent(client_name, client_version),
+        "X-Kolena-Telemetry": str(client_state.telemetry),
```

Comment:
is it possible that we can test the serialized header?

Reply:
I did a manual test with this set to `True/False` and included in the description. Are you referring to adding unit/integ tests for this? From what I saw so far there are no direct testing on these header fields so I only updated the unit tests that stores the corresponding client state

Reply:
yeah, i was referring to add unit/integration tests. i guess integration tests for the round trip should be able to verify this later